pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -584,10 +584,7 @@ def is_local_emulator(self) -> bool:
             return False
         info = self.backend_info
         assert info is not None
-        if info.get_misc("system_type") == "local_emulator":
-            return True
-        else:
-            return False
+        return bool(info.get_misc("system_type") == "local_emulator")
```

Comment:
The cast to `bool` should not be necessary here.

Reply:
Not doing it will make mypy complain